changelog.d/20240319_014508_mann.compi.md

```diff
@@ -0,0 +1,4 @@
+### Security
```

Comment:
The Security category is for security fixes. This PR is not a fix, it's more of a hardening; so I think this entry belongs in the Added category.